lib/libraries/c3/extensions/default-date-format.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(startDate, endDate){
```

Comment:
Ahh, misread this.. yes, c3 supports accepting both strings or functions that return strings as input values, which I think is an extension of the d3 pattern of doing the same when setting attributes and such.. static-vs-dynamic values. It's not a bad pattern, so long as it's consistently used throughout a lib or SDK.